<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>90</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>90</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>200</width>
    <height>90</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>200</width>
    <height>90</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Time</string>
  </property>
  <widget class="QProgressBar" name="progressBar_h">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>118</width>
     <height>13</height>
    </rect>
   </property>
   <property name="maximum">
    <number>15</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="format">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Часов</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Частей</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Долей</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_h">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_c">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_d">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_c">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>118</width>
     <height>13</height>
    </rect>
   </property>
   <property name="maximum">
    <number>143</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="format">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_d">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>118</width>
     <height>13</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1296</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="format">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_cur_time">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
